Content.Shared/Mobs/Systems/MobStateSystem.Subscribers.cs

```diff
@@ -186,5 +188,13 @@ private void OnAttemptPacifiedAttack(Entity<MobStateComponent> ent, ref AttemptP
         args.Cancelled = true;
     }
 
+    private void OnDamageModify(Entity<MobStateComponent> ent, ref DamageModifyEvent args)
+    {
+        if (_damageable.UniversalMobDamageModifier != 1)
+        {
+            args.Damage *= _damageable.UniversalMobDamageModifier;
+        }
```

Comment:
```suggestion
        args.Damage *= _damageable.UniversalMobDamageModifier;
```
I doubt there is any noticeable performance difference from this, so let's do it the simple way